scene/gui/color_picker.cpp

```diff
@@ -35,6 +35,7 @@
 #include "core/math/color.h"
 #include "core/os/keyboard.h"
 #include "core/os/os.h"
+#include "editor/editor_node.h"
```

Comment:
Must #ifdef for TOOLS_ENABLED. Though ideally, the saving and loading palette feature should also work when using the ColorPicker in a game/application project. Possibly with a boolean flag to enable or disable this feature.